lms/djangoapps/teams/tests/test_models.py

```diff
@@ -81,6 +82,48 @@ def test_team_membership_text_type(self):
             "the-user is member of The Team in edx/the-course/1"
         )
 
+@ddt.ddt
+class CourseTeamTest(SharedModuleStoreTestCase):
+    """Tests for the CourseTeam model."""
+
+    @classmethod
+    def setUpClass(cls):
+        from course_modes.models import CourseMode
```

Comment:
Nit: do we need this to be imported locally?  If we do, can you leave a short comment?